tools/get-images.sh

```diff
@@ -2,13 +2,8 @@
 #
 # This script returns list of container images that are managed by this charm and/or its workload
 #
-# static list
-STATIC_IMAGE_LIST=(
-)
 # dynamic list
-git checkout origin/track/2.0
 IMAGE_LIST=()
 IMAGE_LIST+=($(grep "self._metacontroller_image =" src/charm.py | awk '{print $3}' | sort --unique | sed s/,//g | sed s/\"//g))
```

Comment:
Following the latest changes merged to `main`, exposing the oci-image in `metadata.yaml`, we will have to make the following change here:
```suggestion
IMAGE_LIST+=($(find -type f -name metadata.yaml -exec yq '.resources | to_entries | .[] | .value | ."upstream-source"' {} \;))
```

Reply:
What worries me though is that this script won't work for `track/2.0` (the equivalent applies to a couple of our other repos where we recently introduced configurable images). Should we backport this to `track/2.0`, keeping the logic of the initial `get-images-1.7-stable.sh` or do we not need it?

Reply:
@phoevos Initial `get-images-1.7-stable.sh` was never triggered. `track/2.0` has its [own script](https://github.com/canonical/metacontroller-operator/blob/track/2.0/tools/get-images.sh). So, this script applies only to main, it is just git replaced script. If you think it is cleaner to remove non-used script first and then commit new script let's do this, but I think this PR is clear enough.
If you have no reservations please resolve and squash-and-merge this.

Reply:
@phoevos Do not merge it just yet, I will need to address Orfeas concern, to make this script parse configurable images.

Reply:
Won't the above suggestion do?

Reply:
Yes. It will just checked.